packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -177,8 +177,11 @@ export class TabBarRenderer extends TabBar.Renderer {
         );
     }
 
-    createTabId(title: Title<Widget>): string {
-        return 'shell-tab-' + title.owner.id;
+    /**
+     * If tab is part of hidden horz tab bar, add a suffix to keep the id unique across the DOM
+     */
```

Comment:
This documentation does not accurately describe the `createTabId` method, only the behavior of `isPartOfHiddenTabBar`. We can enhance it, and perhaps describe the `@param` instead.